model/integration_test.go

```diff
@@ -25,6 +26,11 @@ func str2ptr(s string) *string {
 	return &s
 }
 
+func str2cyptoptr(s string) *crypto.String {
+	c := crypto.String(s)
+	return &c
+}
+
```

Comment:
do we have units for this function?

Reply:
This is a helper in unit test, it doesn't need unit tests.
It returns a pointer to crypto.String, which has its unit tests.